amqp.go

```diff
@@ -247,6 +252,16 @@ func StopQueueName(invID string) string {
 	return fmt.Sprintf("road-runner-%s-stops-request", invID)
 }
 
+// NewLaunchRequest returns a *JobRequest that has been constructed to be a
+// launch request for the provided job InvocationID.
+func NewLaunchRequest(invocationID string) *JobRequest {
```

Comment:
It was removed in commit 8a8745f63506af9a90024656a3ea30089c79f309, but v2 was branched off that commit's parent, then v3 was branched off v2, and v4 was branched off v3 (because all those changes still required the Job model in the messages).